### PR TITLE
chore: simplify flake8

### PR DESCRIPTION
**Pull request by prd-ha-tran**
# Pull Request Template

## Link to Issue
Paste Jira/Sentry link

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [ ] My code follows the style guidelines of this project. Refer [Olivia Coding Convention](https://paradoxai.atlassian.net/wiki/spaces/OLIVIA/pages/348323898/Coding%2BGuides%2BUpdates)
- [ ] New and existing unit tests pass locally with my changes? Refer: [Olivia API Test Guide](https://paradoxai.atlassian.net/wiki/spaces/OLIVIA/pages/467828752/API+Test+Guide)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] Updated documentation.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,12 +24,9 @@
         flake8      = ">=4.0.1"
         ipython     = ">=8.0.1"
         jedi        = ">=0.18.1"
-        pep8        = ">=1.7.1"
         pep8-naming = ">=0.12.1"
         pre-commit  = ">=2.17.0"
-        pycodestyle = ">=2.8.0"
         pydocstyle  = ">=6.1.1"
-        pyflakes    = ">=2.3.1"
```
Comment (prd-ha-tran) [inline pyproject.toml:32]: flake8 already has those

File: pyproject.toml [hunk 2]
```diff
@@ -42,10 +39,10 @@
 
 
         flake8-django            = "^1.1"
-        flake8-logging-format    = "^0.6"
```
Comment (prd-ha-tran) [inline pyproject.toml:45]: the only thing this plugin gives is enforce `extra` param instead of python string formats and I dont think it's necessary

**Comment by prd-ha-tran** [comment]
@PhuongTMR could you have a look? it's weird that I cant add reviewers

File: pyproject.toml (hunk 1 above)
Comment (PhuongTMR) [reply pyproject.toml:32]: IDK, trust you LOL.

**Review by PhuongTMR — APPROVED** [review]
Thank you!

**Pull request merged**